tests/main/interfaces-openvswitch/task.yaml

```diff
@@ -15,8 +15,14 @@ details: |
     list and deletion.
 
 prepare: |
+    . "$TESTSLIB/pkgdb.sh"
+
     echo "Given openvswitch is installed"
-    apt install -y --no-install-recommends openvswitch-switch
```

Comment:
Good catch, I actually added to support for that to the distro_install_package command but looks like this was dropped during the rebase/master I did